doc/whats_new/v0.22.rst

```diff
@@ -388,6 +388,10 @@ Changelog
   impact when ``metric="precomputed"`` or (``metric="euclidean"`` and
   ``method="exact"``). :issue:`15082` by `Roman Yurchak`_.
 
+- |Efficiency| Parallelize t-SNE using OpenMP prange to compute the
+  gradients.
+  :issue:`13213` by :user:`Thomas Moreau <tommoral>`
+
```

Comment:
t-SNE already has parallel parts since #15082. I'd say 
Improved efficiency of :class:`manifold.TSNE` when ``method="barnes_hut"`` by computing the gradients in parallel.